### DISCUSSION TITLE
Uncaught ReferenceError: require is not defined

### DISCUSSION THREAD

**Discussion by Qix-** [swc-project/swc#1631] [Q&A]
I'm trying to integrate SWC with Rollup by way of `rollup-plugin-swc`, with the intention of outputting an IIFE.

However, no matter which options I pass to SWC, it tries to compile as a CommonJS module - which isn't what I'm using. This results in it trying to load `regenerator-runtime` via `require()` which will never work as I don't have a CommonJS runtime available.

How can I get SWC to forego this requirement and, perhaps, just assume it exists or otherwise import it some other way?

**Comment by kdy1** [comment]
It seems like a bug of swc. Can you file an issue?

**Reply by Qix-** [reply]
Sure thing
